src/dataframerow/show.jl

```diff
@@ -1,7 +1,15 @@
-function Base.show(io::IO, r::DataFrameRow)
-    labelwidth = mapreduce(n -> length(string(n)), max, _names(r)) + 2
-    @printf(io, "DataFrameRow (row %d)", row(r))
-    for (label, value) in pairs(r)
-        print(io, '\n', rpad(label, labelwidth, ' '), something(value, ""))
-    end
+function Base.show(io::IO, dfr::DataFrameRow;
+                   allcols::Bool = !get(io, :limit, false),
+                   splitcols = get(io, :limit, false),
+                   rowlabel::Symbol = :Row)
+    r, c = parentindices(dfr)
+    @printf(io, "DataFrameRow")
+    _show(io, view(parent(dfr), [r], c), allcols=allcols, splitcols=splitcols,
+          rowlabel=rowlabel, summary=false, rowid=r)
 end
+
+Base.show(dfr::DataFrameRow;
+          allcols::Bool = !get(io, :limit, false),
```

Comment:
We use different defaults for the corresponding `AbstractDataFrame` method.

Reply:
right - fixed